src/blurredLocationDisplay.js

```diff
@@ -126,6 +127,12 @@ BlurredLocationDisplay = function BlurredLocationDisplay(options) {
     }
   }
 
+  function defaultPopupDisplay(obj) {
+    var popup_content = "";
+    popup_content += "<h5><a href='" + obj.url + "'>" + obj.title + "</a></h5>";
```

Comment:
As a follow up, let's make a check if `url` and `title` are present in the external API or not to avoid errors.
Can be a FTO, thanks!